ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/StatementExecutor.java

```diff
@@ -223,42 +223,47 @@ private String executeDdlStatement(final PreparedStatement<?> statement, final C
         .get();
   }
 
-  private void handleRunScript(final Command command, final Mode mode) {
+  /**
+   * @deprecated deprecate since 5.2. `RUN SCRIPT` will be removed from syntax in later release.
+   */
+  @SuppressWarnings("DeprecatedIsStillUsed")
+  @Deprecated
```

Comment:
I don't think we can really phase out stuff in the executor, unless we have the semantic that you can't upgrade major versions without redeploying all your queries (which we don't have a solution for today).

Reply:
We don't want gumpf building up in our syntax. If RUN SCRIPT is not supported, then it shouldn't be in the syntax IMHO.

My plan for handling legacy `RUN SCRIPT` commands in the command topic was to add a pre-processor with a simple regex in the statement executor to detect and handle such statements.

 But that's another PR...